client/src/main/java/com/tigrisdata/db/client/search/SearchRequest.java

```diff
@@ -119,10 +132,13 @@ public static final class Builder {
     private SearchFields searchFields;
     private FacetQuery facetQuery;
     private SortOrder sortOrder;
-    private ReadFields fields;
+    private final List<String> includeFields;
+    private final List<String> excludeFields;
 
     private Builder() {
       this.query = QueryString.getMatchAllQuery();
+      this.includeFields = new ArrayList<>();
```

Comment:
should we use LinkedHashSet here instead?

Reply:
That makes sense